EmployeeManagementService/EmployeeManagementService.Domain/Services/EmployeeService.cs

```diff
@@ -273,6 +278,7 @@ private async Task IncrementEmployeeFailedLoginAttempt(EmployeeDB employee)
             }
 
             employee.IsLocked = true;
+            employee.FailedLoginAttempts = (short)attempts;

```

Comment:
Fix bug.

Employee object we pass in was queried before we updated the failed attempts. Therefore, the employee object had the previous FailedLoginAttempt value. We need to set it to newest attempt value before passing in for update.